pkg/util/voljournal.go

```diff
@@ -160,6 +170,36 @@ func (cj *CSIJournal) SetNamespace(ns string) {
 	cj.namespace = ns
 }
 
+// SetTmpNamePrefix updates the temp name in which all temoprary cloned volume
+// starts with. This will help driver to cleanup the stale clone volume
+// created by it.
+// this  will set the temp name as tmpNamePrefix+driverName
+func (cj *CSIJournal) SetTmpNamePrefix(name string) {
+	cj.tmpNamePrefix += name + "-"
+}
+
+// GetTmpNamePrefix returns the temporary name prefix
+/* if first is set
+if the tmpName is csi-tmp-snap-rbd.ceph.csi.com then it will return
+csi-tmp-snap1-rbd.ceph.csi.com
+if  first is not set returns csi-tmp-snap2-rbd.ceph.csi.com
+this will help us storing the snap info on rados and also helps in cleaning the
+stale snaps
+*/
+func (cj *CSIJournal) GetTmpNamePrefix(sufix string, first, isSnap bool) string {
```

Comment:
Might be better to split into two functions -- one for volumes and the other for snaps. Having to pass the `isSnap` param just to pick an entirely different logic path seems odd to me.